### DISCUSSION TITLE
Ability to toggle API key visibility

### DISCUSSION THREAD

**Discussion by Rapptz** [scalar/scalar#1492] [Ideas]
Hello.

Right now you can preset an API key to allow the requests to work seamlessly for users which is a great feature, however one short coming is that since it's always hidden with no toggle there's no good way to let people actually see their keys or copy paste them. It'd be nice if there was a toggle for the `type="password"` forms to allow for this. 

As a workaround, right now I force it to be `type="text"` using some JavaScript.

**Comment by antlio** [comment]
hey @Rapptz! @hwkr added toggling visibility on password field #1599 ✨ does it answer your need?

**Reply by Rapptz** [reply]
Yes! Thanks a lot, this is pretty much exactly as I imagined it.

**Reply by hwkr** [reply]
Awesome! Glad to hear it :)
